### PR TITLE
html forms- Hannah Hoang

### PR DESCRIPTION
**Pull request by hannahhoanggg**
Screenshot 2023-07-18 122141

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      <div>
+        <label for="email">Email</label>
+        <div>
+          <input type="email" id="email" name="user_email" />
+        </div>
+      </div>
+      <div>
+        <label for="coding-select">Why is coding cool?</label>
+        <div>
+          <select id="why-is-coding-cool-select">
```
Comment (shawnkost) [inline html-forms/index.html]: This should be `id="coding-select"` to match your `for` attribute of the `label`

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            <option value="">Select an option</option>
+            <option value="interesting">Because it's interesting</option>
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
+        </div>
+        <div>
+          <label for="favorite-language">Favorite Language (choose one)</label>
```
Comment (shawnkost) [inline html-forms/index.html]: This should just be a `p` tag, because it's not linked to any `input`

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            <input type="radio" id="HTML" name="language" value="HTML" />
+            <label for="HTML">HTML</label>
+            <input type="radio" id="CSS" name="language" value="CSS" />
+            <label for="CSS">CSS</label>
+            <input
+              type="radio"
+              id="JavaScript"
+              name="language"
+              value="JavaScript" />
+            <label for="JavaScript">JavaScript</label>
```
Comment (shawnkost) [inline html-forms/index.html]: `id`s should always be lowercase

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+          </div>
+          <div>
+            <label for="sign-up"
+              >Would you like to sign up for more information?</label
+            >
```
Comment (shawnkost) [inline html-forms/index.html]: This should just be a `p` since it's not associated with any `input`

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+            <div></div>
+            <input type="checkbox" id="sign-me-up" name="sign me up" />
+            <label for="sign-me-up">Sign me up!</label>
+            <div></div>
+            <label for="comments">Comments</label>
+          </div>
+          <textarea cols="30" rows="10"></textarea>
```
Comment (shawnkost) [inline html-forms/index.html]: Needs an `id="comments"`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here Hannah!

File: html-forms/index.html (hunk 5 above)
Comment (hannahhoanggg) [reply html-forms/index.html]: got it!

File: html-forms/index.html [hunk 6]
```diff
@@ -15,36 +15,34 @@ <h1>Coding Form!</h1>
...
-            <label for="sign-up"
-              >Would you like to sign up for more information?</label
-            >
+            <p>Would you like to sign up for more information?</p>
             <div></div>
             <input type="checkbox" id="sign-me-up" name="sign me up" />
             <label for="sign-me-up">Sign me up!</label>
             <div></div>
-            <label for="comments">Comments</label>
+            <label for="comments" id="comments">Comments</label>
```
Comment (shawnkost) [inline html-forms/index.html]: Wrong place. The `id="comments"` needs to go on the `textarea`. The `for` attribute of the `label` matches the `id` of an `input`, `textarea`, etc.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Hannah!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**